tools/pipelines/templates/include-use-node-version.yml

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+# Licensed under the MIT License.
+
+# include-use-node-version template to use a specific node version
+
+parameters:
+# The node version required. The version must already be installed in the image. 
```

Comment:
Hmm does it really? Could we make this task install it if not found? Baking the supported ones into the image still makes sense so they don't have to be downloaded, but installing a not-found one simplifies the scenario @tylerbutler brought up of testing against a new version that might not be already in the image (and would also go back to paying the price of potential flakiness, but that seems acceptable for testing).

Reply:
I'm OK with whatever for expediency, but I would like to reach a conclusion on the Teams thread so we can establish our relative priorities for things like this. If that leads us to want to change this (as I do) then we can follow up with whatever change is needed. The only drawback to that I can think of is that some release branches will be stuck on "old CI config" for a bit if we change it again.

Reply:
How do we enforce the testing part? I could totally see us going back to installing it all the time as opposed to building the image again. For testing, someone can alter this script and install it explicitly.